nightingale_interface/main.py

```diff
@@ -81,15 +89,14 @@ def build(self):
     def init_ros(self):
         # initialize the ros bridge client
         self.client = roslibpy.Ros(
-            "localhost", MovoConfig.Config["RosBridgePort"]
+            MovoConfig.Config["Movo2"]["IP"], MovoConfig.Config["RosBridgePort"]
         )
-        #MovoConfig.Config["Movo2"]["IP"]
         self.client.run()
         asyncio.sleep(0.5)
 
-        self.ros_action_topic = roslibpy.Topic(self.client, "interface/action", "std_msgs/String")
+        self.ros_action_topic = roslibpy.Topic(self.client, "ui/robot/call_action", "std_msgs/String")

```

Comment:
what do these action and interface topics do? I was thinking we would subscribe to a single status topic and make decisions based on the message recieved